### DISCUSSION TITLE
Upcoming Webinar 11/9/2021

### DISCUSSION THREAD

**Discussion by shaselton-usds** [CMSgov/price-transparency-guide#252] [Announcements]
Please join the Centers for Medicare &amp; Medicaid Services (CMS) on Tuesday, November 9,
2021 for the second in a series of technical implementation webinars for the in-network
negotiated rate and out-of-network historical billed charges machine-readable file requirements
in the Transparency in Coverage (TiC) Final Rules (85 FR 72158).

During this second webinar, the CMS team will recap October’s webinar, cover new decisions
arrived on GitHub and provide insight on change process. CMS will also respond to appropriate
questions that are posted in this announcement as well as in the Q&amp;A chat box during the
webinar.

Please register for the webinar [here](https://cms.zoomgov.com/webinar/register/WN_kA6M_5BSRg6rG3xs3knTyA).

Post questions below and like/upvote questions so the team knows what’s most important to you.

**Comment by sforsyth089** [comment]
I posted a question in general discussion that I am hoping for help on: https://github.com/CMSgov/price-transparency-guide/discussions/257. Thank you!  I'll be at the webinar.

**Comment by brbucher** [comment]
Hello, I have a few questions regarding compliance and timelines:

Considering that V1.0 of the schema will be finalized on March 1st;
 Is the expectation that affected organizations will have V1.0 compliant files available by July 1st?
 If not, will there be windows of implementation for the different versions and which version is required for the July 1st date?
 I am concerned that four months is not going to be enough time.

Once the compliance deadline is reached;
 How is regulatory compliance going to be verified?

**Comment by ghost** [comment]
Common question based on multiple posts would be, guidance on the optional use for references in the schema to files/data outside the published file to supplementary files to support duplicate data that can be found specifically in the case of the MRF INN's In-Network Negotiated Rates array elements. This would provide a reduction in storage/egress charges for both the producer and consumer.

**Reply by BhaVem** [reply]
I would also like this addressed soon. This will help with the storage issues and to remove complexity with merging data.

**Comment by tdfow** [comment]
Can CMS address how to address the publication of the negotiated rate when based on percentage of charge scenario?  The schema doesn't support identifying a 'basis charge amount' or identification of percentage of charge, so how should these percentage of charge rates be published? 

Also, would there be a CMS schema update to support this scenario?  
We have heard that some would like to publish the negotiated_rate as null for this situation until guidance is provided, so would that be supported?  Currently, the negotiated rate is required and must represent a dollar amount.  So could there be something in the schema to denote the percentage of charge or charge basis to understand the negotiated rate value that is published for those items/services whose rate is based on percentage of charge?

**Reply by rgouldkc** [reply]
I would also like this addressed. We actually have some provider contracts that are totally based on a discount off billed charges.

**Reply by tdfow** [reply]
@rgouldkc --based on the guidance, if this is important, then you would need to like/upvote questions so the CMS team knows what’s most important to you.

**Comment by perbs** [comment]
I also would like to get some information on the question I posted: #229

**Comment by tdfow** [comment]
Any updates on how the naming standard of the files should be handled after the reporting_plans array was updated in the schema?

**Comment by menckend** [comment]
Hello.  Apologies, as I believe this is slightly out-of-scope for the 11/09 webinar on "_development_ of machine readable files...", but I'm hoping someone might steer me in the right direction.

  I'm wondering where/to-whom I should  look for clarifications on the non-functional requirements associated with the public-disclosure mandate in [§ 147.212](https://www.ecfr.gov/current/title-45/subtitle-A/subchapter-B/part-147/section-147.212).

Specifically, I'm hoping for clarification as to what criteria plans/insurers will be held accountable to in regards to the data-files being "available and accessible."  For example, if the files we generate and host each month are 200TB in size, and the average consumer has a 30mbps Internet connection (which would take 2 years to download one month's data-set), are those files  "available" to the user?  Is there a minimum bandwidth that plans/insurers must have allocated for this access?  Or a minimum number of concurrent downloads that must be supported?  (etc...)

Likewise, hoping to understand what (if any?) compensating controls plans/insurers are permitted to implement to mitigate against financial usage-based attacks.  The "without conditions" clause, has me wondering if we are permitted to block obvious usage-based attacks, or known/suspected bot-nets, etc..

Any assistance greatly appreciated!

**Reply by BDemas** [reply]
I sent a similar question.  I believe we are all concerned about the potential costs of "bad actors."

**Reply by menckend** [reply]
I think that this was touched on briefly during the Q&A section of the webinar, but in a very superficial manner.  What I took away from the response was essentially "you could use a CDN", but that wasn't particularly responsive to my questions.   @shaselton-usds , can you (or anyone else) advise as to the appropriate contact or forum would be to obtain clarification?

**Comment by calfred33** [comment]
Plans are having trouble determining how to proceed with the requirement to disclose providers TINs in the machine-readable files, given that so many providers use their personal social security numbers as TINs. Guidance from CMS on how to handle this issue would be welcome.

**Reply by shaselton-usds** [reply]
I think this addresses your question: https://github.com/CMSgov/price-transparency-guide/discussions/73#discussioncomment-1596247

**Comment by calfred33** [comment]
This does, thank you.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by retjami** [comment]
Would really like to understand CMS position on how plans can protect themselves from excessive egress costs due to bad actors/malicious attacks.

**Comment by Corrigall** [comment]
Can CMS provide a link/location for the recorded webinar on 11/09/2021?
